neon-animated-pages.html

```diff
@@ -86,19 +86,17 @@
 
     },
 
-    observers: [
-      '_selectedChanged(selected)'
-    ],
-
     listeners: {
+      'iron-select': '_onIronSelect',
       'neon-animation-finish': '_onNeonAnimationFinish'
     },
 
-    _selectedChanged: function(selected) {
-
-      var selectedPage = this.selectedItem;
+    _onIronSelect: function(event) {
```

Comment:
Does this event handler also need to handle `iron-deselect`, for cases when we go from a selected item to no selected items?

In the above case, `selected` would be changed, resulting in a callback. However, `iron-select` is only fired when a new item is selected.

Perhaps more generally: does `neon-animated-pages` support having no page selected?